data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/AuthApi.kt

```diff
@@ -43,12 +49,16 @@ class AuthApi(
     }
 
     private suspend fun registerToServer(createdIdToken: String) {
-        httpClient.post<String>("https://ssot-api-staging.an.r.appspot.com/accounts") {
-            headers {
-                set("Authorization", "Bearer $createdIdToken")
+        runCatching {
+            httpClient.post<String>("https://ssot-api-staging.an.r.appspot.com/accounts") {
+                header(HttpHeaders.Authorization, "Bearer $createdIdToken")
+                contentType(ContentType.Application.Json)
+                body = "{}"
```

Comment:
maybe , non need empty body...?

Reply:
When I tried it before, the server returned an error if I didn't send this "{}". If it doesn't seem to work, you can turn it off. 💭 
For the time being, this PR does not have to deal with it.  👍

Reply:
hmm.... OK!